Apromore-Core-Components/Apromore-Manager/src/main/resources/xsd/sanitizeBPMN.xsl

```diff
@@ -31,17 +30,19 @@
 
 <!-- Templates supporting textOnly mode -->
 
-<!-- Inside an element which is disallowed complex content, attributes and text get passed through unchanged. -->
-<xsl:template match="@*|text()" mode="textOnly">
-    <xsl:copy>
-        <xsl:apply-templates select="@*|node()" mode="textOnly"/>
-    </xsl:copy>
+<!-- Inside an element which is disallowed complex content, text gets passed through unchanged. -->
+<xsl:template match="text()" mode="textOnly">
+    <xsl:value-of select="."/>
 </xsl:template>
```

Comment:
Since we know that XML text nodes are leaf nodes, we can do without the recursive machinery.